testing_archive/test_pulse_len_too_low.py

```diff
@@ -34,9 +34,7 @@ def __init__(self):
 
         # changed from 10e3/3->10e3
         decimation_scheme = (DecimationScheme(rates[0], rates[-1]/dm_rates[-1], stages=all_stages))
-        super(TestExperiment, self).__init__(
-            cpid, output_rx_rate=decimation_scheme.output_sample_rate,
-            decimation_scheme=decimation_scheme)
+        super(TestExperiment, self).__init__(cpid, output_rx_rate=decimation_scheme.output_sample_rate)
```

Comment:
```suggestion
        super().__init__(cpid, output_rx_rate=decimation_scheme.output_sample_rate)
```
Same change was made on a different file